medical/gql_mutations.py

```diff
@@ -4,16 +4,18 @@
 
 import django.db.models.base
 import graphene
+from graphene import InputObjectType
 from core import assert_string_length, PATIENT_CATEGORY_MASK_ADULT, PATIENT_CATEGORY_MASK_MALE, \
     PATIENT_CATEGORY_MASK_MINOR, PATIENT_CATEGORY_MASK_FEMALE
```

Comment:
import filter validity
```suggestion
from core import (
    assert_string_length, 
    PATIENT_CATEGORY_MASK_ADULT, 
    PATIENT_CATEGORY_MASK_MALE, 
    PATIENT_CATEGORY_MASK_MINOR, 
    PATIENT_CATEGORY_MASK_FEMALE,
    filter_validity,
```